src/ensembl/src/services/analytics-service.ts

```diff
@@ -1,8 +1,10 @@
 import ReactGA from 'react-ga';
 import { AnalyticsType } from 'src/analyticsHelper';
 
-const googleTrackingID = process.env.GOOGLE_ANALYTICS_KEY
-  ? process.env.GOOGLE_ANALYTICS_KEY
+import config from 'config';
+
+const googleTrackingID = config.googleAnalyticsKey
+  ? config.googleAnalyticsKey
```

Comment:
`const googleTrackingID = config.googleAnalyticsKey || '';`

Reply:
That's done